web/packages/teleport/src/services/discovery/types.ts

```diff
@@ -43,6 +49,30 @@ export type AwsMatcher = {
   integration: string;
   // kubeAppDiscovery specifies if Kubernetes App Discovery should be enabled for a discovered cluster.
   kubeAppDiscovery?: boolean;
+  /**
+   * InstallParams sets the join method when installing on
+   * discovered EC2 nodes
+   */
+  install?: {
+    /**
+     *  EnrollMode indicates the mode used to enroll the node into Teleport.
+     * Valid values: script, eice.
```

Comment:
Does this mean that `Unspecified` is not allowed?
If so, I'd specify the type as `InstallParamEnrollMode.Script | InstallParamEnrollMode.Eice` and remove the comment.

Reply:
I instead removed the `Unspecified` enum value instead